recipes/astrowidgets/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "astrowidgets" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "580328c58b59c7f92e3cb9a301f20293016cb1e258672b1c38487ed3abe820d6"
```

Could you please remove the quotes from the ``sha256``?
```suggestion
  sha256: 580328c58b59c7f92e3cb9a301f20293016cb1e258672b1c38487ed3abe820d6
```